leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -445,6 +449,17 @@ public void showLocations(List<Address> addresses) {
   }
 
   @Override
+  public void showDebouncedLocations(List<Address> addresses) {
+    if (addresses != null) {
+      fillLocationList(addresses);
+      if (!addresses.isEmpty()) {
```

Comment:
What you say sounds interesting but not in the scope of this PR. Please add an issue for that. Many thanks for the suggestion!